File: Indie Python Programming Course/step07/7.04/7.4.02.py
'''

Напишите функцию first_repeated_word , которая принимает строку, состоящую из нескольких слов, слова разделяются между собой пробелом. 
Функция должна найти первое повторяющееся слово и вернуть его в качестве результата. 
Если передана строка, в которой все слова различны, функция first_repeated_word должна вернуть None

Регистр букв при сравнении нужно учитывать

first_repeated_word("ab ca bc ab") => "ab"
first_repeated_word("ab ca bc Ab cA aB bc") => "bc"
first_repeated_word("ab ca bc ca ab bc") => "ca"
first_repeated_word("ab ca bc") => None

Для функции first_repeated_word нужно добавить док-строку  Находит первый дубль в строке

и не забудьте проаннотировать аргументы и возврат функции

Нужно написать только определение функции first_repeated_word 

Sample Input 1:

hello hi hello

Sample Output 1:

hello

Sample Input 2:

hello hi Hello

Sample Output 2:

None

Sample Input 3:

ab ca bc ab

Sample Output 3:

ab

Sample Input 4:

ab ca bc Ab cA aB bc

Sample Output 4:

bc

'''

def first_repeated_word(string):
    'Находит первый дубль в строке'
    string = string.split()
    rep = []
    for i in string:
        if i not in rep:
            rep.append(i)
        else:
            return i
    return None

File: Diving into JavaScript: For Beginners/step07/7.04/7.4.02.js
/*

Пользователь вводит возраст. Напишите программу, которая проверяет, я
вляется ли введенное значение числом и находится ли оно в диапазоне от 18 до 65 лет включительно. 
Если введенное значение не является числом, программа выводит сообщение об ошибке. 
Если число не попадает в диапазон выводится "Доступ запрещен", в иных случаях выводится сообщение "Доступ разрешен".

Sample Input 1:

call me babydoll

Sample Output 1:

Ошибка: Введите ваш возраст

Sample Input 2:

97

Sample Output 2:

Доступ запрещен

Sample Input 3:

18

Sample Output 3:

Доступ разрешен

Sample Input 4:

62

Sample Output 4:

Доступ разрешен

*/

let age = prompt();

if (isNaN(age)) {
    console.log('Ошибка: Введите ваш возраст');
} else {
    if (parseInt(age) < 18 || parseInt(age) > 65) {
        console.log('Доступ запрещен');
    } else {
        console.log('Доступ разрешен');
    }
}
